src/ray/gcs/gcs_server/gcs_worker_manager.cc

```diff
@@ -329,6 +343,21 @@ void GcsWorkerManager::HandleUpdateWorkerNumPausedThreads(
   }
 }
 
+void GcsWorkerManager::Initialize(const GcsInitData &gcs_init_data) {
+  RAY_LOG(INFO) << "Initializing gcs_worker_manager for load dead workers table.";
+  for (const auto &[worker_id, worker_table_data] : gcs_init_data.Workers()) {
+    if (!worker_table_data.is_alive()) {
+      sorted_dead_worker_list_.emplace_back(worker_id, worker_table_data.timestamp());
+    }
+  }
+  // Sort by expiration time, prioritize clearing the history first
+  sorted_dead_worker_list_.sort([](const std::pair<WorkerID, int64_t> &left,
```

Comment:
`const auto& left, const auto& right`

Reason:
- Less typing
- Some containers (i.e. map) has `const` specified as the first item of the pair

Reply:
Thank you for your comment, I do it.